juniper_codegen/src/util/mod.rs

```diff
@@ -4,6 +4,9 @@ pub mod duplicate;
 pub mod parse_impl;
 pub mod span_container;
 
+#[cfg(feature = "tracing")]
+pub mod tracing;
```

Comment:
@ArsileLuci it's better to move it from `util` module to crate root and have a straight-forward `tracing` module.